### PR TITLE
RabbitMQ Bindings - Finish implementation of client side TLS settings

### PR DESCRIPTION
**Pull request by hyperized**
##### SUMMARY
Also implement TLS client settings for POST and DELETE requests in this module.

Currently the module will not honor client TLS settings with these two.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
messaging/rabbitmq_binding

##### ANSIBLE VERSION
```paste below
2.8-devel
```

##### ADDITIONAL INFORMATION
Since the module has been a full rewrite in the past, added an overdue copyright as well.

### CODE REVIEW

**Comment by hyperized** [comment]
@Akasurde I'd appreciate your review as well, but I'm going to take some time in the next few days to add full integration tests to this module since there's none yet.

File: lib/ansible/modules/messaging/rabbitmq/rabbitmq_binding.py [hunk 1]
```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/python
 # -*- coding: utf-8 -*-
 
-# Copyright: (c) 2015, Manuel Sousa <manuel.sousa@gmail.com>
+# Copyright (c) 2015, Manuel Sousa <manuel.sousa@gmail.com>
```
Comment (Akasurde) [inline lib/ansible/modules/messaging/rabbitmq/rabbitmq_binding.py]:
```suggestion
# Copyright: (c) 2015, Manuel Sousa <manuel.sousa@gmail.com>
```

File: lib/ansible/modules/messaging/rabbitmq/rabbitmq_binding.py [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
+# Copyright (c) 2018, Gerben Geijteman <gerben@hyperized.net>
```
Comment (Akasurde) [inline lib/ansible/modules/messaging/rabbitmq/rabbitmq_binding.py]:
```suggestion
# Copyright: (c) 2018, Gerben Geijteman <gerben@hyperized.net>
```

**Comment by Akasurde** [comment]
@mattclay Could you please review this? Thanks.

**Comment by hyperized** [comment]
It's work in progress at this point, I'm still to finish the bindings integration test that utilises TLS.

**Comment by mattclay** [comment]
CI failure in integration tests: https://app.shippable.com/github/ansible/ansible/runs/91510/40/tests

File: test/integration/targets/rabbitmq_lookup/tasks/main.yml [hunk 3]
```diff
@@ -2,4 +2,4 @@
 - include: ubuntu.yml
   when: 
     - ansible_distribution == 'Ubuntu'
-    - ansible_distribution_release != 'trusty'
+    - ansible_distribution_release != 'trusty'
```
Comment (mattclay) [inline test/integration/targets/rabbitmq_lookup/tasks/main.yml]: Keep the line ending here.

**Review by mattclay — CHANGES_REQUESTED** [review]

**Comment by mattclay** [comment]
CI failure in integration tests: https://app.shippable.com/github/ansible/ansible/runs/95070/40/tests

**Comment by gundalow** [comment]
@hyperized Could you please rebase this https://docs.ansible.com/ansible/devel/dev_guide/developing_rebasing.html

**Comment by Akasurde** [comment]
@hyperized What happened ?

**Comment by hyperized** [comment]
@Akasurde got real tired of the tests failing and I didn't feel like I got an answer in #ansible-devel for this. However I realized this PR is doomed anyway because the tests are lacking. I'm going to spend some time in the upcoming holidays to finish the TLS testing ability for the whole of the RabbitMQ spectrum and then resubmit this PR at a later stage.

**Pull request closed without merging**